README.md

```diff
@@ -5,11 +5,12 @@ the [GitHub pages slate theme](https://github.com/pages-themes/slate).
 
 The site is available [here](https://uptane.github.io/).
 
-This repository is a dedicated place for the deployment best practices text. Whatever is on master will be built automatically by github, and deployed at https://uptane.github.io/deployment-considerations.
+This repository is a dedicated place for the deployment best practices text. Whatever is on master will be built automatically by github and deployed at https://uptane.github.io/deployment-considerations.
```

Comment:
Since this commit is devoted to capitalization, please also capitalize  both Deployment Best Practices and Github.

Reply:
Good call, fixed.